### ISSUE TITLE
Search something never mentioned before within page, there's still a searched result poping up

### ISSUE THREAD

**Issue by JackiePengp** [toeverything/AFFiNE#976]
![Image](https://user-images.githubusercontent.com/121489905/218649306-feaa4478-1186-4bdb-b0c1-cd1a0a58ff32.png)

**Comment by JimmFly** [comment]
I think that's expected, it's fuzzy queries. What is your opinion? @darkskygit

**Comment by darkskygit** [comment]
Since indexing performance and memory usage need to be considered in browser, we use the [soundex algorithm](https://en.wikipedia.org/wiki/Soundex) to implement fuzzy indexing. This algorithm is based on character pronunciation compression character data. When indexing Latin characters, it can significantly reduce memory usage and ensure that the search results are correct, but searching for characters with similar pronunciation can also get the same results.

In simple terms, this is the expected result from the fuzzy search feature.
